ui/scripts/instanceWizard.js

```diff
@@ -1094,8 +1094,15 @@
 
             var userdata = args.data.userdata;
             if (userdata != null && userdata.length > 0) {
+
+                // NOTE code duplicated from sanatizeReverse in ui/utils.js because of call propagation issues
+                userdata = userdata
+                            .replace(/&amp;/g, "&")
+                            .replace(/&lt;/g, "<")
+                            .replace(/&gt;/g, ">");
```

Comment:
Nicolas reported he had a test case where the calling variity did work, I so I'll experiment more to see if I can get it to work and replace above with a call to cloudstack.sanitizeReverse(userdata);